crates/core/tedge_api/src/alarm.rs

```diff
@@ -145,6 +150,47 @@ impl ThinEdgeAlarm {
             ))
         }
     }
+
+    pub fn new_try_from(
```

Comment:
I assume this function is meant to replace `ThinEdgeAlarm::try_from` so the current name is temporary? If so, we'll need to remember to rename it when removing `ThinEdgeAlarm::try_from`, although we could perhaps use a different name than `try_from`, as usually it refers to `TryFrom::try_from`, so it can be very confusing.

Reply:
Thats right, while replacing the existing ones, this needs to be renamed as `try_from`.